### ISSUE TITLE
ErrorException after upgrade to 6.4.2

### ISSUE THREAD

**Issue by cvantum** [Intermesh/groupoffice#364]
Hello,
i did a upgrade from 6.3.69 to 6.4.2 with your Repository.
Server:
- Debian Stretch
- Apache 2.4.25
- PHP 7.0.33
- Maria-Server 10.1.37

Apt-Upgrade on *CLI* went fine. But first start of Web-Page gave me an error:
```
ErrorException in /usr/share/groupoffice/go/core/orm/Property.php at line 438: Trying to get property of non-object
```
I opened the upgrade-Page (`[URL]/groupoffice/install/upgrade.php`)  for a DB-Upgrade and i could start the process. But then i ran into a next error:

```
ErrorException in /usr/share/groupoffice/go/core/Module.php at line 160: Trying to get property of non-object
PDOException in /usr/share/groupoffice/go/core/orm/Property.php at line 899: SQLSTATE[HY000]: General error: 1364 Field 'digest' doesn't have a default value, referer: http://[URL]
```
I saw the login-screen at start-page and could login. But now every menu-item is missing. I just see the background-color of our installation.

Is it possible to restart the upgrade-process manually on command-line or to see why i got an error while upgrading my installation?

Best regards,

**Comment by cvantum** [comment]
A little update by myself. I cloned a machine and cleaned the installation via `dpgk-reconfigure groupoffice` and started the wizard for a new installation.

After i entered credentials for first admin-account i got a new error-message:

```
Exception in /usr/share/groupoffice/go/core/db/Utils.php at line 21: SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes on query (67) ALTER TABLE `core_search` ADD INDEX(`keywords`) at 2019-04-12T05:21:08+00:00
```

**Comment by mschering** [comment]
This will be fixed by 6.3.4
